experimental/dispatch_profiler/library.py

```diff
@@ -6,6 +6,7 @@
 
 import enum, re
 from enum import auto
+from ml_dtypes import bfloat16
```

Comment:
The dispatch_profiler changes could be there own PR, right?
IIUC, IREE should still support BF16 but it will promote them to FP32. Which is fine for landing the dispatch profiler changes.